### PR TITLE
Closes https://github.com/coala/coala/issues/3389

### LINKED ISSUE #3389: Change default_coafile naming convention.
Should we change the name of `default_coafile` in `coalib` to `system_coafile`?
It will avoid confusion due to the following lines in `Constants.py`
```
system_coafile = os.path.join(coalib_root, 'default_coafile')

user_coafile = os.path.join(os.path.expanduser('~'), '.coarc')

default_coafile = '.coafile'
```

### PR DESCRIPTION
**Pull request by LeafyLi**
default_coafile: Rename to system_coafile

Closes #3389

### CODE REVIEW

**Comment by LeafyLi** [comment]
I think i fixed it, but how do i rerun the tests?

**Comment by Makman2** [comment]
just repush to rerun the tests. But apparently you haven't fixed it yet ;)

**Comment by nemaniarjun** [comment]
Use git grep to find all instances of the name. :)

File: coalib/misc/Constants.py [hunk 1]
```diff
@@ -109,11 +109,11 @@
                                     'languages',
                                     'definitions')
 
-system_coafile = os.path.join(coalib_root, 'default_coafile')
+system_coafile = os.path.join(coalib_root, 'system_coafile')
 
 user_coafile = os.path.join(os.path.expanduser('~'), '.coarc')
 
-default_coafile = '.coafile'
+system_coafile = '.coafile'
```
Comment (nemaniarjun) [inline coalib/misc/Constants.py]: dont make this change

**Comment by nemaniarjun** [comment]
Sorry, I screwed this up really bad,
You are supposed to do only two changes, 
You need to change the name of the file in coalib from `default_coafile` to `system_coafile` 
and in constants.py on line 112. 
thats it.
I got confused in the same confusion we are trying to remove by this PR.
Again, sorry :)

**Comment by nemaniarjun** [comment]
Also your commit shortlog should say `default_coafile` instead of `default_coalib`

**Pull request closed without merging**